notebooks/data-sources/TestGrid/metrics/time_to_fail.ipynb

```diff
@@ -0,0 +1,1180 @@
+{
```

Comment:
<p>The docstring feels lonely when it's outside the scope of the function :D</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/337/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>We can bring it inside :D </p>

Reply:
yes updated :+1: